src/main/java/seedu/eduke8/common/EditableList.java

```diff
@@ -0,0 +1,8 @@
+package seedu.eduke8.common;
+
+public interface EditableList {
+
+    public void add(Displayable displayable);
+
+    public Displayable delete(int index);
+}
```

Comment:
Good thinking, this interface will allow us to differentiate functionality more cleanly between certain lists that are purely for display/access, vs others that the users might wish to affect. Nice implementation!